The Ultimate Customer Segmentation Analysis (Data Epic).ipynb

```diff
@@ -0,0 +1,1649 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    class Segment():
</pre><p>You didn't have to create a class here. It's an unnecessary complication. aim for simplicity in your code.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Data-Epic/data-wrangling-Idris-Lawal/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>